### ISSUE TITLE
Archivist: indexed save game data properties

### ISSUE THREAD

**Issue by BigBadBleuCheese** [Llama-Logic/PlumbBuddy#19]

**Comment by amethystlilac** [comment]
Ideal minimum:
* Last played household
* Last played world and lot
* Save name (from the game and/or within PB)
* Number of save slot options that you could load
* Original date created
* Thumbnail

**Comment by BigBadBleuCheese** [comment]
Keep in mind that this is per individual save file. A lot of the things you've listed appear to be chronicle aggregates. Maybe you intend them to be both, though.

**Comment by amethystlilac** [comment]
Yes, I meant per save file. You look at a save file that you may want to load or just browse through, and you see those things at the time it was saved in the chronicle, because that can determine if *this* is the option you want to load.

Nice to have:
* List of played households (including sims in those households). Including world would be even better. (When did Virginia move to Henford-on-Bagley? Oh, in this save slot. When did Olivia die? Must have been the save before this, because she's nowhere to be found.)
* If new save, auto saved, manual save, or save-as (not sure we can easily determine save vs. save-as vs new save).

**Comment by amethystlilac** [comment]
* Photos in the save at that point.
* Available thumbnails from the save at that point.
* Search by traits. That could honestly help in a lot of situations with bugged sims in addition to being nice to know, AND it covers a lot of other things. Want to know all the occults in your save? Search the trait.
* Traits for the last active household's sims.
* Inventory per sim, household, and object. Same reason, bugged things.
* Game version when saved.
* state of `enable_autogeneration_same_sex_preference` because I'm *still* insulted two months later, preferably with the ability to change it.
* Money per household (more because of a game bug that goes back years and isn't likely to ever be fixed than anything else, but also nice to know).
* Season.
* Days played.
* Scheduled Drama Nodes, because the game will never not have drama node bugs. No idea what to do with the information, but I'm sure someone could figure something out.

I think that's most of the most useful things?

**Comment by amethystlilac** [comment]
From Black Cherry: Eco Footprint per neighbourhood

**Comment by BigBadBleuCheese** [comment]
My emphasis added:
> * state of `enable_autogeneration_same_sex_preference` because I'm _still_ insulted two months later, *preferably with the ability 

Altering items is out of scope for Archivist with the sole exceptions of thumbnails and timestamps, because they are for identification purposes.

**Comment by amethystlilac** [comment]
Updated list, with a goal of *archiving* saves instead of searching them, when you're looking at a specific save in a chronicle:

- Save name (from the game and/or within PB)
- Thumbnail
- Last played household, with which sims, on which lot, and in which world
- List of played households at the time of save
- Season
- Days played
- Time of save (real time; current at time of saving)
- Game version when saved
- Mods when saved and mod changes relative to current mods folder
- Packs when saved and pack changes relative to current pack list
- Number of other save slot options that you could load for that save chronicle, possibly something like 'This is the 52nd save for the 100 Baby Challenge Save chronicle, out of 77 available saves'
